nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -130,7 +130,7 @@ let
           ${concatStringsSep " \\\n" ([ "-f qcow2" ]
```

Comment:
this should probably just be using `lib.escapeShellArgs`

Reply:
Would that not again lead to the `$NIX_DISK_IMAGE` shell variable being put in between single quotes and prevent shell expansion in the resulting script, similar to [this previous suggestion](https://github.com/NixOS/nixpkgs/pull/229845/commits/996db6d059cd8e6f3b5815a33816bf8ea16f42b5#r1184961872)?